src/d2r-map.ahk

```diff
@@ -107,6 +117,9 @@ unitHwnd1 := WinExist()
 offsetAttempts := 6
 ticktock := 0
 While 1 {
```

Comment:
Should we still respect the 'leftMargin'' and 'topMargin' from the settings? So we would have `window left margin + left margin = actual margin`?

Reply:
we could and probably should, though levelx and level y are pseudo margins in the mean time as they are persistent across sessions. I was gonna add a statement to set the map as docked or undocked.